src/components/pages/LandingPage.jsx

```diff
@@ -2,92 +2,169 @@ import { useTranslation } from 'react-i18next'
+            </p>
+          ))}
+          <div className='grid py-8 place-content-center'>
+            <img alt='tabs' src={TabImage}></img>
+          </div>
 
-          <ul>
-            <li>Once an hour, the Insight Agent will capture the programs and websites on the computer.</li>
-            <li>This information will be sent by internet to our servers.</li>
-            <li>All of the information will be collected with other information from other participants around the United States.</li>
-            <li>The servers will sort the information by type into categories like social media, entertainment, work productivity, education and more.</li>
+          <ul className='px-10'>
+            {firstUnorderedListParagraphs.map((p, index) => (
+              <li className='py-2 md:py-6' key={index}>
+                {bullet}
+                {p}
+              </li>
+            ))}
           </ul>
 
-          <h3>Your Privacy is Important</h3>
+          <div className='font-extrabold text-center py-10 text-xl md:text-2xl lg:text-3xl'>
+            <h3>Your Privacy is Important</h3>
+          </div>
 
-          <ul>
-            <li>We use industry standard practices to safeguard your data.</li>
-            <li>Additionally, your data will be anonymized before we use it for research.</li>
-            <li>You are in control and can opt out of the study at any time. </li>
+          <ul className='px-10'>
+            {privacyList.map((p, index) => (
+              <li className='py-2 md:py-6' key={index}>
+                {bullet}
+                {p}
+              </li>
+            ))}
           </ul>
+          <div className='grid py-8 place-content-center'>
+            <img alt='tabs' src={RightTaskbar}></img>
+          </div>
 
-          <h3>What will happen to the data:</h3>
+          <div className='font-extrabold text-center py-10 text-xl md:text-2xl lg:text-3xl'>
+            <h3>What will happen to the data:</h3>
+          </div>
 
-          <ul>
-            <li>We will share our findings with technology builders so that they can build technology that’s useful for people like you and your community.</li>
-            <li>We want to make sure that all customers are represented.</li>
+          <ul className='px-10'>
+            {whatWillHappenList.map((p, index) => (
+              <li className='py-2 md:py-6'>
+                {bullet}
+                {p}
+              </li>
+            ))}
           </ul>
 
-          <p>
-            { "You are the expert of you and we'll pay you for that expertise. We'll compensate you at the beginning of the study and at the conclusion of the study." }
-          </p>
-
-          <p>
-            { "We'll also ask you for additional information through voluntary surveys and interviews throughout the study period." }
-          </p>
+          <div className='pt-10'>
+            <p>
+              You are the expert of you and we'll pay you for that expertise.
```

Comment:
I didn't map over every p/li tag - just the ones that had the same repeated style and were more verbose.